tests/testutils/test_configuration_test.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from pathlib import Path
+
+from pytest import LogCaptureFixture
+
+from pylint.testutils.configuration_test import get_expected_output
+
+HERE = Path(__file__).parent
+USER_SPECIFIC_PATH = HERE.parent.parent
+DATA_DIRECTORY = HERE / "data"
+
+
+def test_get_expected_output(caplog: LogCaptureFixture) -> None:
+    caplog.set_level(logging.INFO)
+    exit_code, _ = get_expected_output(DATA_DIRECTORY / "t.toml", USER_SPECIFIC_PATH)
+    assert "Too much .out files" in str(caplog.text)
+    assert exit_code == -1
+    exit_code, _ = get_expected_output(DATA_DIRECTORY / "u.toml", USER_SPECIFIC_PATH)
+    assert exit_code == -1
+    assert "Wrong format for .out file name" in str(caplog.text)
+    exit_code, _ = get_expected_output(DATA_DIRECTORY / "v.toml", USER_SPECIFIC_PATH)
```

Comment:
We might actually want another error here: `.toml doesn't exists`. At least for now I think we should add `v.toml` to `tests/testutils/data`.

Reply:
7794ba5 . The functional test loops on all the toml/cfg/ini file so the toml is guaranteed to exists if we launch ``get_expected_output``.

Reply:
Ah of course!